INSTALLATION.rst

```diff
@@ -328,7 +328,7 @@ environment manager.
 
 .. attention::
 
-   :term:`PySparse` and :term:`FiPy` are not presently included in Canopy,
```

Comment:
1. [Pysparse](http://pysparse.sourceforge.net) is not consistent with [PySparse's](http://pysparse.sourceforge.net/introduction.html) own spelling.

1. CamelCaseRocks

QED. Existing spelling is correct

Reply:
1. I disagree, [PySparse](http://pysparse.sourceforge.net/) is not consistent with [Pysparse](http://pysparse.sourceforge.net/introduction.html)'s own spelling.

  | Count | PySparse | Pysparse |
  | ---       | ---             | ---             |
  | [index](http://pysparse.sourceforge.net/index.html) | 0 | 13 |
  | [intro](http://pysparse.sourceforge.net/introduction.html) | 5 | 8 |
2. Take that up with the [Pysparse devs](https://sourceforge.net/projects/pysparse/support).

Reply:
It's Pysparse in headings and PySparse in sentences. Everyone knows that. It's important that we get this one correct.

Reply:
I thought it was PySparse when sunny, Pysparse when overcast?

Reply:
There are no PySparse devs

Reply:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) suggests the correct name is "pysparse." When used as a proper noun, English grammar prefers "Pysparse."